src/components/board/SharingTabSidebar.vue

```diff
@@ -194,6 +198,38 @@ export default {
 		clickDeleteAcl(acl) {
 			this.$store.dispatch('deleteAclFromCurrentBoard', acl)
 		},
+		clickTransferOwner(newOwner) {
+			OC.dialogs.confirmDestructive(
+				t('deck', 'Are you sure you want to transfer the board {title} for {user} ?', { title: this.board.title, user: newOwner }),
+				t('deck', 'Transfer the board.'),
+				{
+					type: OC.dialogs.YES_NO_BUTTONS,
+					confirm: t('deck', 'Transfer'),
+					confirmClasses: 'error',
+					cancel: t('deck', 'Cancel'),
+				},
+				async(result) => {
+					if (result) {
+						try {
+							this.isLoading = true
+							await this.$store.dispatch('transferOwnership', {
+								boardId: this.board.id,
+								newOwner,
+							})
```

Comment:
`transferOwnership` action has been called with two params: `boardId` and `newOwner`, but the action expects 3 params: `boardId`, `newOwner` and `owner`.